CYLGame/www/static/player.js

```diff
@@ -407,8 +416,13 @@ class Player {
 
   drawCurFrame() {
     if (this.cur_frame >= 0 && this.cur_frame < this.replay_frames.length) {
-      this.playback_progress_bar.css("width", (this.cur_frame / (this.replay_frames.length - 1)) * 100 + "%");
-      this.playback_progress_bar_text.html("Frame " + (this.cur_frame + 1) + " of " + (this.replay_frames.length));
+      if (this.cur_frame == 1) {
```

Comment:
Is there a better place for this instead of hardcoding for the first frame? Maybe in the function that sets `this.reply_frames`?

Reply:
That makes sense, since the length should change only when new frames are loaded